normits_demand/models/notem/production_models.py

```diff
@@ -25,7 +25,7 @@
 
 # local imports
 import normits_demand as nd
-
+import logging
```

Comment:
Remove this import if its not being used now

Reply:
removed